catalog/USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/COVER

```diff
@@ -0,0 +1,23 @@
+'gee:skip_indexing': true,
+local id = 'USGS/NLCD_RELEASES/2019_REL/RCMAP/V5/COVER';
+local subdir = 'USGS/NLCD_RELEASES';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+
+local template = import 'RCMAP.libsonnet';
+
+local basename = std.strReplace(id, '/', '_');
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+template + {
+id: id,
```

Comment:
please add
 'gee:skip_indexing': true,
 (this is a workaround for the fact we don't yet accept dataset images)

Reply:
That is on line 1. Is that the correct place for it?

Reply:
Added as line 15

Reply:
Removed from line 1, have on line 13